wordpress-mobile/WordPress-iOS
# Move WordPress.com account services

To keep our codebase consistent, I've moved all WordPress.com services that were part of `WordPressComApi` into a service class of its own named `WordPressComServiceRemote`.

**To test:**
1. Launch the app
2. In the login screen, tap on "Create Account"
3. Fill the form
4. Make sure the account and blog are created successfully

/cc @jleandroperez
